doc/dev/crimson/crimson.rst

```diff
@@ -207,20 +131,12 @@ The following options can be used with ``vstart.sh``.
     Valid types include ``HDD``, ``SSD``(default), ``ZNS``, and ``RANDOM_BLOCK_SSD``
     Note secondary devices should not be faster than the main device.
 
-To start a cluster with a single Crimson node, run::
-
-  $  MGR=1 MON=1 OSD=1 MDS=0 RGW=0 ../src/vstart.sh \
-    --without-dashboard --bluestore --crimson \
-    --redirect-output
 
-Another SeaStore example::
+To start a simple cluster with a single core Crimson OSD, run::
 
-  $  MGR=1 MON=1 OSD=1 MDS=0 RGW=0 ../src/vstart.sh -n -x \
+  $  MGR=1 MON=1 OSD=1 MDS=0 RGW=0 ../src/vstart.sh -n \
     --without-dashboard --seastore \
-    --crimson --redirect-output \
-    --seastore-devs /dev/sda \
-    --seastore-secondary-devs /dev/sdb \
-    --seastore-secondary-devs-type HDD
+    --crimson --redirect-output
```

Comment:
@cyx1231st, @xuxuehan - Is anything more should be mentioned here for a basic vstart seastore cluster?

Reply:
LGTM.

Is MGR mandatory to start a basic cluster? I was setting it to 0.

Reply:
I think so, the monitor relies on the manager to exist for additional motioning info. This should also result in an health warning.

Reply:
Seems ` --nodaemon` is required for `--redirect-output`.